packages/core/test/interpreter/onerror.js

```diff
@@ -26,6 +26,52 @@ test('saga onError is optional', assert => {
   })
 })
 
+test('saga passes thrown Error instance in onError handler', assert => {
```

Comment:
isnt this test a duplicate of "saga onError is called for uncaught error"?

Reply:
didn't notice 👍 
changed test case